imgix/urlbuilder.py

```diff
@@ -59,6 +62,11 @@ def __init__(
                           'Use `include_library_param` instead.',
                           DeprecationWarning, stacklevel=2)
 
+        if isinstance(domains, (list, tuple)) and (len(domains) > 1):
```

Comment:
@jayeb Something worth noting, with `lens(domains) > 1` here it would still be valid to pass in a single domain via list or tuple, eg `UrlBuilder(['sherwinski.assets.net'])`
We can add logic later to convert the object back to a string or we can restrict any lists/tuples of any length outright. Thoughts?

Reply:
I think when the time comes to actually remove sharding functionality, we should not allow the domain to be specified in a list. However, since this PR is concerned only with adding deprecation warnings, I think we should still allow it.